src/containers/DidDocumentView/DidDocumentView.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import { Contact } from '../../types/Contact'
+import './DidDocumentView.scss'
+import Code from '../../components/Code/Code'
+
+type Props = {
+  didDocument: object
+  children: any
```

Comment:
Question:
I know you're using the children, but do you need to explicitly list them as prop?

Reply:
Yes, if I remove it. I receive an error.

`"Type '{ children: string | undefined; didDocument: object; }' is not assignable to type 'IntrinsicAttributes & Props'.\n  Property 'children' does not exist on type 'IntrinsicAttributes & Props'."`

Tried to understand why it happens, but I wasn't able too. However, adding the children etc works.

Reply:
OK got it, good that you checked; it's because you're using the children, so TypeScript needs to explicitely know about it.